docs/nodes/build/launch-an-avalanche-validator-on-aws-with-one-click.md

```diff
@@ -0,0 +1,359 @@
+</iframe>
+
+
+## Introduction
+
+Avalanche is an open-source platform for launching decentralized applications
+and enterprise blockchain deployments in one interoperable, highly scalable
+ecosystem. Avalanche is the first decentralized smart contracts platform built
+for the scale of global finance, with near-instant transaction finality.
+
+With the intention of enabling developers and entrepreneurs to on-ramp into the
+Avalanche ecosystem with as little friction as possible, Ava Labs recently
+launched an offering to deploy an Avalanche Validator node via the AWS
+Marketplace. This tutorial will show the main steps required to get this node up
+and running and validating on the Avalanche Fuji testnet.
+
+## Product Overview
+
+The Avalanche Validator node is available via [the AWS
+Marketplace](https://aws.amazon.com/marketplace/pp/prodview-nd6wgi2bhhslg).
+There you'll find a high level product overview. This includes a product
+description, pricing information, usage instructions, support information and
+customer reviews. After reviewing this information you want to click the "Continue to Subscribe" button.
+
+## Subscribe to This Software
+
+Once on the "Subscribe to this Software" page you will see a button which
+enables you to subscribe to this AWS Marketplace offering. In addition you'll
+see Terms of service including the seller's [End User License
+Agreement](https://awsmp-offer-legal-documents-prod-iad.s3.us-east-1.amazonaws.com/prod/arn%3Aaws%3Acatalog%3Aus-east-1%3A%3Aoffer/f2norleeteb3ui74y2aypa0r2/00be9c1c-b4bc-41fa-bef8-2c77d8b28639?X-Amz-Security-Token=FwoGZXIvYXdzEFkaDF2Wax%2F6GPKJo5NMfCKIAmGr6m4cdM0bZ9eenwTY9xri8lgWonpiIqABvO3tfAOTJhA%2B5m979V9JitosI3I%2BCb7A95OGiSet9UecTvkseLyQr%2FALfG78KY%2BmHn3wiO%2FKFapCeyvygxZNYtNOH6hIJivYQ4agK262tU707kHLDXwHFZR42BPSKZXpB6vtDxmD3yu%2BO6lwc3B0qqyRyDKyJlKwsiPIahdxO90HMvYQZx5OMWpM0ZqH2Htr%2B43UR8hHov9h2CJKVJ4L1PlQgbvlIyxCXBbyjEpiNL8BNy9K%2FzLip%2FAjoyBtl7FASvGikQ4WLwa4NcHHZzh1BoTcHNH2z6boRDgGVR0Rmwu4cshml2GxYoHPKeL77SiyyfmdBjJPQ8jPLez0%2B%2Fd41mp8JshMsIYXHuNFM%2B2hSazwrPcEWre8xA%2FxlL2IDx2Lm0n3LEjYmjqJc6NZFHcuXo2LLi9Kf0jvretBj3dHOADL2Y4oXA%3D%3D&X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Date=20230111T072642Z&X-Amz-SignedHeaders=host&X-Amz-Expires=86399&X-Amz-Credential=ASIASPPEMYX2LJWCIFGD%2F20230111%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Signature=e441c3fd4a801666587b7cde4868f92025d265879f8a0955f4c7040b1f771f53)
+and the [AWS Privacy Notice](https://aws.amazon.com/privacy/). After reviewing
+these you want to click on the "Continue to Configuration" button.
+
+## Configure This Software
+
+This page lets you choose a fulfillment option and software version to launch
+this software. No changes are needed as the default settings are sufficient.
+Leave the `Fulfillment Option` as `64-bit (x86) Amazon Machine Image (AMI)`. The
+software version is the latest build of [the AvalancheGo full
+node](https://github.com/ava-labs/avalanchego/releases), `v1.9.5 (Dec 22, 2022)`, AKA `Banff.5`. 
```

Comment:
```suggestion
node](https://github.com/ava-labs/avalanchego/releases), `v1.9.5 (Dec 22, 2022)`, AKA `Banff.5`.  This will always show the latest version.
```

Reply:
Resolved